### ISSUE TITLE
In "Components of SWIS" section, the default section do not have underline

### ISSUE THREAD

**Issue by manuignatius** [subnero1/subnero1.github.io#231]
It works in SWAN page, but not in swis page. See below.

<img width="1328" alt="Screenshot 2022-11-08 at 2 37 58 PM" src="https://user-images.githubusercontent.com/4144181/200522570-627253bc-bd7d-46da-8e79-9993bdca2a60.png">

<img width="1412" alt="Screenshot 2022-11-08 at 2 38 38 PM" src="https://user-images.githubusercontent.com/4144181/200522635-e6ac0fd5-9862-4abc-9a10-bb4d61461c78.png">

**Comment by notthetup** [comment]
I think the `id`s matter here. I think we changed the ID names from the SWAN version but we need to also change the `for` and `name`.

![SCR-20221108-t4s](https://user-images.githubusercontent.com/553140/200570911-a0d31d3e-7d50-4ab5-9532-b72f82d3dcfc.png)
![SCR-20221108-t60](https://user-images.githubusercontent.com/553140/200571174-0a1345be-5248-439c-91b1-bf940a59f005.png)

**Comment by notthetup** [comment]
Figured it out. There's appreantly specific styles to do this in the CSS.. Ugh!!

```
#SwanBot:checked ~ .type-container #swan-tab-one,
#SwanCloud:checked ~ .type-container #swan-tab-two,
#SwanNode:checked ~ .type-container #swan-tab-three,
#all-products:checked ~ .type-container #tab-one,
#L5Lseries:checked ~ .type-container #tab-two,
#M25Mseries:checked ~ .type-container #tab-three,
#Accessories:checked ~ .type-container #tab-four {
    border-bottom:2px solid #E6782F;
}

#SwanBot:checked ~ .tab-panels #swan-panel-one,
#SwanCloud:checked ~ .tab-panels #swan-panel-two,
#SwanNode:checked ~ .tab-panels #swan-panel-three,
#all-products:checked ~ .tab-panels #panel-one,
#L5Lseries:checked ~ .tab-panels #panel-two,
#M25Mseries:checked ~ .tab-panels #panel-three,
#Accessories:checked ~ .tab-panels #panel-four {
  display:block
}
```


I'll think of a more generic way to do this.

**Comment by notthetup** [comment]
Fixed in 3e08e6520b1088ea14ce1dfec29e82402272985e
